.ci/Jenkinsfile

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env groovy
+@Library('apm@current') _
+
+pipeline {
+  agent { label 'metal || darwin || master' }
+  options {
+    timeout(time: 2, unit: 'HOURS')
+    buildDiscarder(logRotator(numToKeepStr: '100', artifactNumToKeepStr: '30', daysToKeepStr: '30'))
+    timestamps()
+    ansiColor('xterm')
+    disableResume()
+    durabilityHint('PERFORMANCE_OPTIMIZED')
+    rateLimitBuilds(throttle: [count: 60, durationName: 'hour', userBoost: true])
+    quietPeriod(10)
+  }
+  stages {
+    /**
+     Checkout the code and stash it, to use it on other stages.
+    */
+    stage('Generate Docs Link') {
+      options { skipDefaultCheckout() }
+      steps {
+        deleteDir()
+        script {
+            def PRnum = sh(script: "echo ${CHANGE_URL} | cut -d / -f7", returnStdout: true)
+            githubPrComment(message: "Docs preview will be available at: https://observability-docs_${PRnum}.docs-preview.app.elstc.co/guide/en/observability/current/index.html")
```

Comment:
`CHANGE_BRANCH` env variable is the one generated on the fly if using the default checkout, see https://apm-ci.elastic.co/pipeline-syntax/globals#env

Therefore I'd suggest to `options { skipDefaultCheckout() }` be removed